docs/security-considerations.rst

```diff
@@ -159,17 +160,21 @@ Sending and Receiving Ether
 
 - If you want to send Ether using ``address.transfer``, there are certain details to be aware of:
 
-  1. If the recipient is a contract, it causes its fallback function to be executed which can, in turn, call back the sending contract.
-  2. Sending Ether can fail due to the call depth going above 1024. Since the caller is in total control of the call
-     depth, they can force the transfer to fail; take this possibility into account or use ``send`` and make sure to always check its return value. Better yet,
-     write your contract using a pattern where the recipient can withdraw Ether instead.
-  3. Sending Ether can also fail because the execution of the recipient contract
-     requires more than the allotted amount of gas (explicitly by using ``require``,
-     ``assert``, ``revert``, ``throw`` or
-     because the operation is just too expensive) - it "runs out of gas" (OOG).
-     If you use ``transfer`` or ``send`` with a return value check, this might provide a
-     means for the recipient to block progress in the sending contract. Again, the best practice here is to use
-     a :ref:`"withdraw" pattern instead of a "send" pattern <withdrawal_pattern>`.
+  1. If the recipient is a contract, it causes its receive or fallback function
+     to be executed which can, in turn, call back the sending contract.
+  2. Sending Ether can fail due to the call depth going above 1024. Since the
+     caller is in total control of the call depth, they can force the
+     transfer to fail; take this possibility into account or use ``send`` and
+     make sure to always check its return value. Better yet, write your
+     contract using a pattern where the recipient can withdraw Ether instead.
+  3. Sending Ether can also fail because the execution of the recipient
+     contract requires more than the allotted amount of gas (explicitly by
+     using ``require``, ``assert``, ``revert``, ``throw`` or because the
```

Comment:
`throw` does not exist anymore. Also the first sentence does not really match the list of these functions - `revert` explicitly fails and does not run out of gas.

Reply:
And as this is an area that can confuse people, again, maybe link to https://solidity.readthedocs.io/en/v0.5.14/control-structures.html#error-handling-assert-require-revert-and-exceptions

Reply:
@chriseth Yeah, actually I'm not sure I get the connection between `Sending Ether can also fail because the execution of the recipient contract requires more than the allotted amount of gas` and `using require, assert, revert`.

Reply:
Ah ok, if it explicitly asserts/requires something on the gas amount

Reply:
So I would keep it like that (just removing `throw`)